modules/drip.h

```diff
@@ -0,0 +1,55 @@
+/* Drip
+ 
+   Imitates the sound of dripping water via Physical Modeling Synthesis.
+   
+   Credits
+   Original author(s) : Perry Cook
+   Year: 2000
+   Ported from soundpipe by Ben Sergentanis, May 2020
+*/
```

Comment:
Move this down to above line 20 so that it documents the Drip class. Otherwise we get a warning for ` warning: Compound daisysp::Drip is not documented.` from Doxygen.